lib/sidekiq/cron/job.rb

```diff
@@ -194,21 +194,21 @@ def self.load_from_hash! hash
       #   }
       # ]
       #
-      def self.load_from_array array
+      def self.load_from_array(array, options = {})
         errors = {}
         array.each do |job_data|
-          job = new(job_data)
+          job = new(job_data.merge(options.with_indifferent_access))
```

Comment:
Do we need `with_indifferent_access` here?

Reply:
Just noticed that we are transforming the keys into strings.

https://github.com/sandip-mane/sidekiq-cron/blob/f9335c89c92fa4f7872e2789de978fb8c1988e3d/lib/sidekiq/cron/job.rb#L273

I will revert this.